src/app/app.module.ts

```diff
@@ -173,6 +174,7 @@ const APP_PROVIDERS = [
   AuthService,
   DatePipe,
   FlagService,
+  FullScreenService,
```

Comment:
@eslawski I think I added the service here?

Reply:
ahh gotcha. Not sure how I missed that one!